.gitignore

```diff
@@ -38,3 +38,6 @@ docs/html/*
 operators/hack/deployer/deployer
 operators/hack/deployer/config/run-config.yml
 
+# ignore CI config files
+operators/run-config.yml
```

Comment:
`deployer-config.yml` would be more explicit

Reply:
That's a good idea, but I'd prefer to do it in separate PR, WDYT?

Reply:
Sounds good to me